packages/stable/src/types.ts

```diff
@@ -209,9 +209,17 @@ export interface ApiKeyRequest {
   serviceAccountId: CogniteInternalId;
 }
 
+export type ArrayPatchString =
+  | { set: Set<string> }
+  | { add?: Set<string>; remove?: Set<string> };
+
 export type ArrayPatchLong =
-  | { set: number[] }
-  | { add?: number[]; remove?: number[] };
+  | { set: Array<number> }
+  | { add?: Array<number>; remove?: Array<number> };
+
+export type ArrayPatchClaimNames =
+  | { set: Array<ClaimName> }
+  | { add?: Array<ClaimName>; remove?: Array<ClaimName> };
```

Comment:
```suggestion
  | { add?: ClaimName[]; remove?: ClaimName[] };
```
we use short syntax in other places